testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationMemberTest.java

```diff
@@ -233,86 +232,4 @@ public void testDeleteGroupOnOrganizationRemoval() {
         assertFalse(testRealm().groups().groups().stream().anyMatch(group -> group.getName().startsWith("kc.org.")));
     }
 
-    @Test
```

Comment:
We should probably keep this test.

Reply:
Agree, why was it going to be removed?

Reply:
I probably messed something up during rebase/merge and threw out the test. Will fix